packages/components/psammead-media-player/src/index.jsx

```diff
@@ -60,6 +67,14 @@ CanonicalMediaPlayer.propTypes = {
   showPlaceholder: bool,
   src: string.isRequired,
   skin: oneOf(['classic', 'audio']),
+  service: string.isRequired,
+  mediaInfo: shape({
+    title: string.isRequired,
+    datetime: string,
```

Comment:
lines 73-75 should be required :)

Reply:
> "Default props are only shallowly merged."
https://stackoverflow.com/questions/38123498/how-do-you-provide-default-props-for-nested-shape-in-react

Ah, if these arnt infact required then this is fine as is :)